eslint.config.mjs

```diff
@@ -9,6 +9,14 @@ import globals from 'globals'
 
 export default [
   'eslint:recommended',
+  // Ignored dirs
```

Comment:
since we're using the config.mjs file, do we need the env variable in package json? seems like the docs imply that if we have this file we don't need to use that flag

Reply:
It tries to parse as YAML without the env var, will be fixed and we can remove it when flat config replaces the previous approach